model/host/host.go

```diff
@@ -119,6 +119,11 @@ type Host struct {
 	DockerOptions DockerOptions `bson:"docker_options,omitempty" json:"docker_options,omitempty"`
 }
 
+type HostsByDistro struct {
```

Comment:
This is only used by your aggregation, so at the very least it should be private. Though I would make it an anonymous struct within the aggregation, unless you plan on using it elsewhere

Reply:
As discussed in person - it is used outside its package.